client/src/modules/Feedback/actions/thunks.js

```diff
@@ -0,0 +1,32 @@
+import { addFeedback } from '../../../feedbackManager';
+import { getFormData } from '../selectors';
+import { validation_rules } from '../validation/rules';
```

Comment:
'validation_rules' Is it a consistent kind of naming?

Reply:
Ohhh, ill change it on 'rules'